### DISCUSSION TITLE
Full body image generation

### DISCUSSION THREAD

**Discussion by PraveenRepswal** [lllyasviel/Fooocus#2179] [Q&A]
When I try to generate image with Fooocus then it automatically crops the head of image if I restrict it to generate full body image including head.

This are the input images
![Input images](https://github.com/lllyasviel/Fooocus/assets/129259520/367ae8f3-08ab-40b8-8aa6-7f2f251e9ecf)

This is the output
![Output](https://github.com/lllyasviel/Fooocus/assets/129259520/e4051e6e-437d-4913-8034-322ed0ece910)

Settings
![Settings](https://github.com/lllyasviel/Fooocus/assets/129259520/39477c72-16be-483a-8c74-3fbc137b74ad)

Is there any special prompt to generate complete body including complete head

**Comment by poor7** [comment]
Try using 704×1408

**Reply by PraveenRepswal** [reply]
But I want the image in 1024*1024

**Reply by poor7** [reply]
Then you will have to make the second photo also square and preferably with a resolution of 1024x1024.

**Comment by blckbx** [comment]
Also give these negative prompts a try: cropped, out of frame

**Comment by gin063** [comment]
Try to describe the hair style.

**Comment by shake** [comment]
I use fooocus 2.5.5, default setting,  only change the  Proportion

create 10 picture, 

prompts：

full body shot of a Beautiful girl,  smiling, beach background, stand on the beach，Barefoot

1024*1024:   only 3 pic is full body

768:1408 : 9 pic is full body.

**Comment by shake** [comment]
Full body shot of a Beautiful girl, smiling, stand on the beach, barefoot.
 
try it, if you choose high quality ， better。

**Comment by shake** [comment]
Full body shot 

Full-body shot of a  Beautiful girl, wavy hair,smiling，big natural breast，slim body，black trousers，wearing high heels, looking at viewer.

Full-body shot of a  Beautiful girl, wavy hair,smiling，big natural breast，slim body，black trousers，wearing high heels, looking at viewer, Urban Chic, Edgy, New York City, spring, blooming flowers, Central Park, iconic skyline, Empire State and Chrysler Buildings.

Full body shot of a Beautiful girl, smiling, stand on the beach, barefoot.
